paket.dependencies

```diff
@@ -1,35 +1,59 @@
 source http://nuget.org/api/v2
+redirects: on
+strategy: max
 
-nuget FSharp.Compiler.Service
 nuget Fantomas
+nuget FSharp.Compiler.Service
+nuget FSharp.Core
+nuget FSharp.Management
 nuget FSharp.ViewModule.Core
-nuget FsXaml.Wpf
-nuget SourceLink.SymbolStore 0.5.0
 nuget FSharpLint.Core prerelease
-nuget VSSDK.IDE.Deploy.14
-nuget FSharp.Management
 nuget FsPickler
-nuget FSharp.Core redirects:on
+nuget FsXaml.Wpf
+nuget SourceLink.SymbolStore 0.5.0
 
+group vssdklegacy
+    source http://nuget.org/api/v2
+    framework >= net40
+    
+    nuget Microsoft.VisualStudio.Shell.Interop.8.0
+    nuget Microsoft.VisualStudio.Shell.Immutable.10.0
+    nuget VSSDK.ComponentModelHost.11
+    nuget VSSDK.DTE
+    nuget VSSDK.DTE.8
+    nuget VSSDK.DTE.10
+    nuget VSSDK.NavigateTo
+    nuget VSSDK.VSLangProj
+    
+    
+group vssdk
+    source http://nuget.org/api/v2
+    framework >= net45
+    
+    nuget Microsoft.VisualStudio.Editor
+    nuget Microsoft.VisualStudio.Language.Intellisense
```

Comment:
Sorry not as familiar with Paket.  But one item here that stands out is the lack of version.  It's imperative that Visual Studio editor dependencies include version information.  The lack of versions + VS SDK installation + MSBuild resolution rules can easily lead to a project depending on a version of the VS editor binaries that it did not intend to.  Run into this many times myself. 

Can these be tied to a specific version of the binaries?  In particular i believe the editor ones should be 14.0.  


Due to the VS SDK installation + MSBuild resolution rules